pkg/csi/cinder/driver.go

```diff
@@ -83,21 +85,30 @@ type DriverOpts struct {
 	ClusterID string
 	Endpoint  string
 	PVCLister v1.PersistentVolumeClaimLister
+
+	WithTopology bool
```

Comment:
can we group it with the endpoint and clusterID members?

Reply:
Done